x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_gaps/trial_license_complete_tier/configs/serverless.config.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { createTestConfig } from '../../../../../../config/serverless/config.base';
+
+export default createTestConfig({
+  testFiles: [require.resolve('..')],
+  junit: {
+    reportName: 'Detection Engine - Rule Gaps Integration Tests - Serverless Env - Complete Tier',
+  },
+  kbnTestServerArgs: [
+    `--xpack.securitySolution.alertIgnoreFields=${JSON.stringify([
+      'testing_ignored.constant',
+      '/testing_regex*/',
+    ])}`, // See tests within the file "ignore_fields.ts" which use these values in "alertIgnoreFields"
+    `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+      'bulkCustomHighlightedFieldsEnabled',
+      'alertSuppressionForEsqlRuleEnabled',
```

Comment:
do we need these feature flags for rule gaps tests?

Reply:
nope, deleted it